src/rgw/rgw_sal_motr.cc

```diff
@@ -1668,6 +1664,9 @@ int MotrObject::MotrReadOp::prepare(optional_yield y, const DoutPrefixProvider*
   source->set_obj_size(ent.meta.size);
   source->category = ent.meta.category;
   *params.lastmod = ent.meta.mtime;
+  for (auto& it : source->get_attrs()){
```

Comment:
We should not print this here, this can become very large.

Reply:
I think its left over from debugging

Reply:
If this is required the used debug level 90. and remove the special chars and use __func__ instead of function name.

Reply:
I have removed this log, this should not have been in the PR in the first place. I missed this, my bad.